src/app/app.routing.ts

```diff
@@ -32,8 +32,10 @@ const appRoutes: Routes = [
   { path: 'setting', component: SettingsComponent },
   { path: 'services-health', component: ServicesHealthComponent, canActivate: [AuthGuard] },
   { path: 'service-discovery', component: ServiceDiscoveryComponent },
+  // /user-management
   { path: 'user-management', loadChildren: './components/core/user-management/user.management.module#UserManagementModule' },
-  { path: 'user-profile', component: UserProfileComponent, canActivate: [AuthGuard] },
+  // user/profile
+  { path: 'user', loadChildren: './components/core/user-management/user.management.module#UserManagementModule' },
```

Comment:
Path for both `user-management` and `user/profile` are same (i.e. `./components/core/user-management/user.management.module#UserManagementModule`). Cann't it be possible to use only one?

Reply:
This is only to make a clean URL and already discussed. We are good with this

Reply:
Ok